lib/logstash/inputs/syslog.rb

```diff
@@ -118,6 +119,7 @@ def server(protocol, output_queue)
   rescue => e
     if !stop?
       @logger.warn("syslog listener died", :protocol => protocol, :address => "#{@host}:#{@port}", :exception => e, :backtrace => e.backtrace)
+      @metric_errors.increment(:listener_exception)
```

Comment:
I think @ph mentioned elsewhere an idea to namespace exceptions, and I agree.

I don't think we should call them exceptions: we should call them errors. Thoughts @ph and @purbon?

`{ "errors" { "listener" => 123 } }`